### PR TITLE
Fix <abi_ver> in platform tuple for Visual Studio

### PR DESCRIPTION
**Pull request by t-sommer**
and order versions from low to high

see also comments by @traversaro in #516

### CODE REVIEW

File: docs/2_3_fmu_distribution.adoc [hunk 1]
```diff
@@ -130,29 +130,26 @@ ABI version `<abi_ver>`::
...
+|Visual Studio 2013 (MSVC++ 12.0)
 
-|90
-|Visual Studio 9.0 (2008)
+|140
+|Visual Studio 2015 (MSVC++ 14.0)
 
-|80
-|Visual Studio 8.0 (2005)
+|150
```
Comment (traversaro) [inline docs/2_3_fmu_distribution.adoc]: If this is the toolset version, it should be 141 for VS2017, or perhaps I am missing something?

Comment (t-sommer) [reply docs/2_3_fmu_distribution.adoc]: Fixed it. Thanks!

**Comment by beutlich** [comment]
I wonder why

- the platform names clarified for FMI 2.0.1 (#302) are no longer valid, i.e., why this break in compatibility is acceptable?
- the separation of shared objects (e.g., DLL) and static objects (e.g., static LIB) was abandoned? In FMI 2.0.1 the static objects are always located (optionally) in the corresponding platform directory of the shared object.

**Comment by beutlich** [comment]
> 
> 
> I wonder why
> 
>     * the platform names clarified for FMI 2.0.1 (#302) are no longer valid, i.e., why this break in compatibility is acceptable?
> 
>     * the separation of shared objects (e.g., DLL) and static objects (e.g., static LIB) was abandoned? In FMI 2.0.1 the static objects are always located (optionally) in the corresponding platform directory of the shared object.

Any comments?

**Pull request merged**